src/main/java/hachi/Hachi.java

```diff
@@ -0,0 +1,410 @@
+        System.out.print("[" + statusIcon + "] " + toDelete.getName());
+        System.out.println(" has been deleted from your list.");
+
+        taskArrayList.remove(taskNumber - 1);
+    }
+
+    private static int getDeleteTaskNumber(String cleanedInput) throws HachiException {
+        int indexOfTaskNum = cleanedInput.indexOf("DELETE") + 6; // find index of task number
+        int taskNumber = 0;
+
+        try {
+            taskNumber = Integer.parseInt(cleanedInput.substring(indexOfTaskNum).trim()); // parse string to int
+        } catch (NumberFormatException e){
+            HachiException.checkOutOfBounds(indexOfTaskNum);
+        }
+
+        return taskNumber;
+    }
+
+    private static int getMarkTaskNumber(String cleanedInput) throws HachiException {
+        int indexOfTaskNum = cleanedInput.indexOf("MARK") + 4; // find index of task number
+        int taskNumber = 0;
+
+        try {
+            taskNumber = Integer.parseInt(cleanedInput.substring(indexOfTaskNum).trim()); // parse string to int
+        } catch (NumberFormatException e){
+            HachiException.checkOutOfBounds(indexOfTaskNum);
+        }
+
+        return taskNumber;
+    }
+
+    /**
+     * Prints to the console a goodbye message for the user.
+     */
+
+    public static void printGoodbyeMessage() {
+        System.out.println("\tGoodbye! Hope you have a marvelous day.");
+    }
+
+    private static void loadFile(ArrayList<Task> taskArrayList) throws FileNotFoundException, HachiException {
+        File taskFile = new File("src/main/java/hachi.txt");
+        Scanner s = new Scanner(taskFile);
+
+        while (s.hasNext()) {
+            String[] line = s.nextLine().split(" \\| ");
+            Task toAdd;
+
+            switch (line[0]){
+            case "T":
+                toAdd = new Todo(line[2]);
+                break;
+
+            case "D":
+                toAdd = new Deadline(line[2], line[3]);
+                break;
+
+            case "E":
+                toAdd = new Event(line[2], line[3], line[4]);
+                break;
+
+            default:
+                throw new HachiException(HachiException.CORRUPTED_SAVE_MESSAGE);
+            }
+
+            if (line[1].equals("X")) {
+                toAdd.setCompleteness(true);
+            }
+            taskArrayList.add(toAdd);
+        }
+    }
+
+    private static void save(ArrayList<Task> taskArrayList) throws IOException {
+        try {
+            FileWriter fw = new FileWriter("src/main/java/hachi.txt", false);
+            fw.write("");
+            fw.close(); // to clear text file
+        } catch (IOException e) {
+            System.out.println("\n\tCreating new task list save...");
+        }
+
+        FileWriter fw = new FileWriter("src/main/java/hachi.txt");
+        for (Task task : taskArrayList) {
+            if (task != null) {
+                fw.write(task.getSaveFormat() + "\n");
+            } else break;
```

Comment:
The else loop body should be wrapped by curly brackets irrespective of how many lines there are in the body ! so e.g. 
else {
    break;
}